PR_DESCRIPTION_TEMPLATE.md

```
@@ -0,0 +1,71 @@
+# PR Title (squash title)
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)